.buildkite/pipeline.yml

```
@@ -461,60 +461,87 @@ steps:
   - group: "EDMFX"
     steps:
 
-      - label: ":genie: explicit baroclinic wave (ρe) equilmoist with EDMFX"
-        command:
-          - julia --color=yes --project=examples examples/hybrid/driver.jl --job_id sphere_baroclinic_wave_rhoe_equilmoist_edmfx --initial_condition MoistBaroclinicWaveWithEDMF --ode_algo SSP33ShuOsher --apply_limiter false --moist equil --precip_model 0M --turbconv edmfx --edmfx_sgs_flux true --dt 1secs --t_end 6mins --dt_save_to_disk 30secs --reference_job_id sphere_baroclinic_wave_rhoe_equilmoist
-        artifact_paths: "sphere_baroclinic_wave_rhoe_equilmoist_edmfx/*"
+      - label: ":genie: Diagnostic EDMFX test in a box"
+        command: >
+          julia --color=yes --project=examples examples/hybrid/driver.jl --job_id diagnostic_edmfx_test_box
+          --turbconv diagnostic_edmfx
+          --moist equil
+          --config box --hyperdiff true --kappa_4 1e8
+          --x_max 1e4 --y_max 1e4 --x_elem 2 --y_elem 2 --z_elem 45 --z_max 30e3 --dz_bottom 30
+          --dt 1secs --t_end 100secs
+          --dt_save_to_disk 10secs
+        artifact_paths: "diagnostic_edmfx_test_box/*"
         agents:
           slurm_mem: 20GB
 
-      - label: ":genie: EDMFX Bomex on a sphere"
-        command:
-          - julia --color=yes --project=examples examples/hybrid/driver.jl --job_id edmfx_bomex_sphere --ode_algo SSP33ShuOsher --apply_limiter false --initial_condition Bomex --moist equil --turbconv edmfx --subsidence Bomex --edmf_coriolis Bomex --ls_adv Bomex --surface_setup Bomex --z_elem 60 --z_stretch false --z_max 3e3 --dt 0.001secs --t_end 0.1secs --dt_save_to_disk 0.01secs --reference_job_id edmf_bomex
-        artifact_paths: "edmfx_bomex_sphere/*"
+      - label: ":genie: Diagnostic EDMFX Bomex in a box"
+        command: >
+          julia --color=yes --project=examples examples/hybrid/driver.jl --job_id diagnostic_edmfx_bomex_box
+          --initial_condition Bomex --subsidence Bomex --edmf_coriolis Bomex --ls_adv Bomex --surface_setup Bomex
+          --turbconv diagnostic_edmfx --edmfx_entr_detr true
```

```suggestion
          --turbconv diagnostic_edmfx
```
(We don't have entrainment for diagnostic edmfx in the main yet, so this argument won't do anything. Maybe delete it to be clear?)